src/parent/actions/parent.js

```diff
@@ -6,3 +6,23 @@ export function close(windowName) {
         type: ACTION_TYPES.CLOSE
     };
 }
+
+function dragOut(code) {
+    return {
+        type: ACTION_TYPES.DRAG_OUT,
+        code
+    };
+}
+
+export function dragAccept() {
+    return {
+        type: ACTION_TYPES.DRAG_ACCEPT
+    };
+}
+
+export function favouriteDroppedOutside(code, position) {
+    return (dispatch) => {
+        dispatch(dragOut(code));
+        window.opener.parentService.createChildWindow(null, position);
```

Comment:
This feels a bit weird to me and I'm not to keen on using `window` and the `parentService` like this. This might actually be a good use case for using events/the inter-application messaging bus. Let's discuss.